lib/tasks/alchemy/sitemap.rake

```diff
@@ -0,0 +1,95 @@
+class SiteMapIssueFinder
+  attr_reader :save
+  attr_accessor :depth_issue_page_ids, :lft_rgt_issue_page_ids
+
+  def initialize(save: false)
+    @save = save
+    @depth_issue_page_ids = Set.new
+    @lft_rgt_issue_page_ids = Set.new
+  end
+
+  def depth_issues
+    puts "Find Page Depth Issues"
```

Comment:
Adding a new line here would look nicer in log with lots of other output (deprecation notices, etc)